```sh
[nix-shell:~/.cache/nixpkgs-review/pr-346260]$ results/devbox/bin/updater
panic: lstat go.jetpack.io/devbox/examples: no such file or directory

goroutine 1 [running]:
main.main()
	go.jetpack.io/devbox/testscripts/testrunner/updater/main.go:19 +0x2b
```

Looks like `subPackages = [ "cmd/devbox" ];` should be added.

--- a/pkgs/development/tools/devbox/default.nix
+++ b/pkgs/development/tools/devbox/default.nix
@@ -2,16 +2,17 @@
 , fetchFromGitHub
 , installShellFiles
 , lib
+, stdenv
 }:
 buildGoModule rec {
   pname = "devbox";
-  version = "0.13.0";
+  version = "0.13.3";
 
   src = fetchFromGitHub {
     owner = "jetpack-io";
     repo = pname;
     rev = version;
-    hash = "sha256-gTJotzg7ot9C7NVDiauVJHpkSXBwaK7arjtnwUGFqgE=";
+    hash = "sha256-A1dl5bQ7+Qe+MxzplQ2duGqWwgZYHd6M2MQUPPFlx14=";
   };
 
   ldflags = [
@@ -20,6 +21,8 @@ buildGoModule rec {
     "-X go.jetpack.io/devbox/internal/build.Version=${version}"
   ];
 
+  subPackages = [ "cmd/devbox" ];
+
   # integration tests want file system access
   doCheck = false;
 
@@ -27,7 +30,7 @@ buildGoModule rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  postInstall = ''
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
     installShellCompletion --cmd devbox \
       --bash <($out/bin/devbox completion bash) \
       --fish <($out/bin/devbox completion fish) \

